src/modules/Thread/components/ParentMessageInfo/ParentMessageInfoItem.tsx

```diff
@@ -263,7 +263,7 @@ export default function ParentMessageInfoItem({
               <div className="sendbird-parent-message-info-item__thumbnail-message__icon-wrapper__icon">
                 <Icon
                   type={isVideoMessage(message) ? IconTypes.PLAY : IconTypes.GIF}
-                  fillColor={IconColors.GRAY}
+                  fillColor={IconColors.ON_BACKGROUND_2}
```

Comment:
Is it okay to use this in `dark theme`?

Reply:
yep
<img width="424" alt="Screen Shot 2023-09-22 at 4 03 10 PM" src="https://github.com/sendbird/sendbird-uikit-react/assets/16806397/449faf12-457f-45b1-9aed-c310d51ea42d">

Reply:
https://sendbird.slack.com/archives/C057B72HN84/p1695369631720649?thread_ts=1695367522.168849&cid=C057B72HN84
Fixed by https://github.com/sendbird/sendbird-uikit-react/pull/757